src/compiler/crystal/semantic/top_level_visitor.cr

```diff
@@ -1025,7 +1025,11 @@ class Crystal::TopLevelVisitor < Crystal::SemanticVisitor
 
   def check_ditto(node)
     stripped_doc = node.doc.try &.strip
-    if stripped_doc == ":ditto:" || stripped_doc == "ditto"
+    # TODO: keep only `:ditto:` after 0.30.0
+    if stripped_doc == "ditto"
+      raise "`ditto` is deprecated, use `:ditto:` instead"
+      node.doc = @last_doc
```

Comment:
this line is pointless when raising, ditto below.

Reply:
What do you mean by failing silently? Don't handle the case at all?